e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/aiclassification/AIClassificationEMTestBase.kt

```diff
@@ -13,9 +14,13 @@ import org.evomaster.core.search.EvaluatedIndividual
 import org.evomaster.e2etests.spring.openapi.v3.SpringTestBase
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertTrue
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 
 abstract class AIClassificationEMTestBase : SpringTestBase(){
 
+    private val log : Logger = LoggerFactory.getLogger(InputEncoderUtilWrapper::class.java)
```

Comment:
why is this logger referring to `InputEncoderUtilWrapper`? and not for example `AIClassificationEMTestBase`?